api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -46,30 +46,40 @@
           + " ORDER BY NULL")
   Set<DbUser> findAllUsersWithAuthoritiesAndPageVisits();
 
-  /** Find users matching the user's name or email */
+  /** Find users matching the user's name or username */
   @Query(
-      "SELECT dbUser FROM DbUser dbUser WHERE dbUser.dataAccessLevel IN :dals "
-          + "AND ( lower(dbUser.username) LIKE lower(concat('%', :term, '%')) "
+      "SELECT dbUser FROM DbUser dbUser "
+          + "WHERE lower(dbUser.username) LIKE lower(concat('%', :term, '%')) "
           + "OR lower(dbUser.familyName) LIKE lower(concat('%', :term, '%')) "
-          + "OR lower(dbUser.givenName) LIKE lower(concat('%', :term, '%')) )")
-  List<DbUser> findUsersByDataAccessLevelsAndSearchString(
-      @Param("dals") List<Short> dataAccessLevels, @Param("term") String term, Sort sort);
+          + "OR lower(dbUser.givenName) LIKE lower(concat('%', :term, '%'))")
+  List<DbUser> findUsersBySearchString(@Param("term") String term, Sort sort);
```

Comment:
called by `UserService.findUsersBySearchString()` which now comes in 2 versions: see changes in UserServiceImpl.